### PR TITLE
Add `openai-aca-lb` template

### PR DESCRIPTION
**Pull request by andredewes**
https://github.com/Azure-Samples/openai-aca-lb

### CODE REVIEW

**Comment by andredewes** [comment]
@hemarina Done! 

Here's the instructions: https://github.com/Azure-Samples/openai-aca-lb/blob/main/README.md#configuring-the-openai-endpoints

**Comment by v-xuto** [comment]
@hemarina Sure. We will test this template today.

**Comment by v-xuto** [comment]
@hemarina, @andredewes  We have finished this template test, and we filed one issue. Please review.
- https://github.com/Azure-Samples/openai-aca-lb/issues/2

**Comment by andredewes** [comment]
> @hemarina, @andredewes We have finished this template test, and we filed one issue. Please review.
> 
> * [The dotnet command running path is not specified Azure-Samples/openai-aca-lb#2](https://github.com/Azure-Samples/openai-aca-lb/issues/2)

@hemarina  Fixed!

**Comment by v-xuto** [comment]
@hemarina [Issue#2](https://github.com/Azure-Samples/openai-aca-lb/issues/2) has been fixed and there are currently no issues with this template.

File: website/static/templates.json [hunk 1]
```diff
@@ -1367,5 +1367,22 @@
...
+  {
+    "title":"Azure OpenAI priority-based load balancer with Azure Container Apps",
+    "description":"A load balancer crafted specifically for Azure OpenAI, which is aware of 'retry-after' headers and intelligently pause traffic when certain OpenAI endpoints reach maximum capacity.",
+    "preview":"./templates/images/openai-aca-loadbalancer.png",
+    "website":"https://github.com/Azure-Samples",
+    "author": "Andre Dewes",
+    "source":"https://github.com/Azure-Samples/openai-aca-lb",
+    "tags":[
+      "aca",
+      "containerapps",
```
Comment (hemarina) [inline website/static/templates.json]: aca is for Azure Container App. 
```suggestion
```

File: website/static/templates.json [hunk 2]
```diff
@@ -1367,5 +1367,22 @@
...
+      "openai",
```
Comment (hemarina) [inline website/static/templates.json:1397]:
```suggestion
      "openai",
      "ai",
```

File: website/static/templates.json [hunk 3]
```diff
@@ -1367,5 +1367,22 @@
...
+      "msft",
+      "bicep",
+      "csharp",
```
Comment (hemarina) [inline website/static/templates.json]:
```suggestion
      "dotnetCsharp",
```

**Comment by andredewes** [comment]
@hemarina Done!

**Pull request merged**